spacy_llm/tasks/util/serialization.py

```diff
@@ -39,20 +39,20 @@ def set_cfg(self, cfg: Dict[str, Any]) -> None:
         for key, value in cfg.items():
             setattr(self, key, value)
 
-    def get_examples(self) -> Optional[List[Dict[str, Any]]]:
+    def get_prompt_examples(self) -> Optional[List[Dict[str, Any]]]:
```

Comment:
We shouldn't have called this `get_examples` in https://github.com/explosion/spacy-llm/pull/152 because typically we have a `get_examples: Callable[[], Iterable["Example"]]` in `initialize()` which works on spaCy's `Example` objects. So I suggest to be explicit everytime we mean "prompt" example, to avoid confusion.